app/views/welcome/index.html.erb

```diff
@@ -1 +1,42 @@
-<h1>Here's a welcome page...</h1>
\ No newline at end of file
+<h1>Here's a welcome page...</h1>
+<div>
+  <p>
+    Our charity works to collect clothing, furniture, food, toys, books, hygiene products, and other miscellaneous materials for the homeless &amp; people in need.
+  </p>
+  <p>
+    WE ACCEPT ALL KINDS OF DONATIONS TO HELP THOSE IN NEED!
+  </p>
+  <ul>
+    <li>
+      Donations of food, hygiene products ,&nbsp;clothing, &amp;&nbsp;furniture &nbsp; are especially appreciated by the less fortunate
```

Comment:
Please change &s to and. In some cases they should be removed completely. For instance line 11 should be "products, clothing and furniture" 

Please change all caps to downcase.

line 11 needs a period.

line 14 is an unfinished thought.

21 recycling and redistributing 

please go through and check for proper capitalization like the words New and Used on line 21.

Please remove iframe and video.

remove everything after 501c3 line.